Simlar/src/SMLRLinphoneHandler.m

```diff
@@ -478,7 +495,7 @@ - (AVAudioSessionCategoryOptions)generateAudioSessionCategoryOptions
         case SMLRCallStatusEnded:
             return 0;
         case SMLRCallStatusIncomingCall:
-            return AVAudioSessionCategoryOptionMixWithOthers|AVAudioSessionCategoryOptionDuckOthers|AVAudioSessionCategoryOptionAllowBluetooth;
```

Comment:
Yes. However, we have to test it with and without bluetooth because of Issue #25.